packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -33,10 +33,6 @@ const ConfirmWithLink = ({
   resendStatus,
   errorMessage,
 }: ConfirmWithLinkProps) => {
-  // (temporarily?) removed `Open With Webmail` functionality
```

Comment:
We are planning to entirely remove the "confirm with link" option, so will not need to add this functionality